packages/image_picker/image_picker/CHANGELOG.md

```diff
@@ -1,9 +1,10 @@
-## NEXT
+## 1.0.8
 
 * Updates minimum supported SDK version to Flutter 3.13/Dart 3.1.
 * Updates support matrix in README to indicate that iOS 11 is no longer supported.
 * Clients on versions of Flutter that still support iOS 11 can continue to use this
   package with iOS 11, but will not receive any further updates to the iOS implementation.
+* Bumps min version of iOS part to start with version with privacy manifest included.
```

Comment:
I'm having trouble parsing this (I'm not sure what it means for the minimum version of something to start with something means). How about:

> Updates minimum iOS implementation version to include a privacy manifest.

(This applies throughout the PR.)